project/settings.py

```diff
@@ -227,8 +227,13 @@
     'django.contrib.gis',
     'django.contrib.staticfiles',
     'mapit',
+    'raven.contrib.django.raven_compat',
 ]
 
+RAVEN_CONFIG = {
```

Comment:
I think we need to override the `environment` here too, if the python client works the same as rails.

In the Ruby config we have this line to set the environment to integration/staging/production, instead of `RAILS_ENV` (always in production):

https://github.com/alphagov/govuk_app_config/blob/9d9bb0adba6f479e8a9b93adba7099b1a5f11ade/lib/govuk_app_config/configure.rb#L7

Reply:
Thanks, I've added that now.